### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by rth**
* Make the example dataset deterministic
 * Use scikit-learn instead of sklearn in the requirements (see https://github.com/scikit-learn/scikit-learn/issues/8215)
 * Add missing dependency [future](https://pypi.python.org/pypi/future)

### CODE REVIEW

**Comment by johannfaouzi** [comment]
Thanks for the pull request. I'm going to run the README with the deterministic dataset so that the figures match the code then merge your pull request.

**Comment by rth** [comment]
Maybe it could be worth changing the random seed, I'm not sure which one produces prettier graphs.
